src/js/components/molecules/filter-category/filter-category.scss

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+@import "~scss/settings/_variables.scss";
+
+.filter-category {
+    text-align: center;
+}
+
+.filter-category__filter-list li {
```

Comment:
There is no styling for `.filter-category__filter-list` so it's just there to be part of the selector for a child `<li>`. Not sure about that.

`.filter-category__list` and `filter-category__list-item` come to mind… the latter would also allow the use of `&:first-of-type` instead of having to use `:first-child` on line 15.

Reply:
thanks; done.